src/test/java/com/levchenko/hw8/CalculateTest.java

```diff
@@ -0,0 +1,62 @@
+package com.levchenko.hw8;
+
+import org.junit.Before;
+import org.junit.jupiter.api.Assertions;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class CalculateTest {
+
+    private final Calculate calculate = new Calculate();
+    private UserInput userInput;
+
+    @Before
+    public void setUp() {
+        userInput = Mockito.mock(UserInput.class);
+
+    }
+
+
+    @Test
+    public void numsSum() {
+        Mockito.when(userInput.getFirstNum()).thenReturn(2.0);
+        Mockito.when(userInput.getSecondNum()).thenReturn(3.0);
+        Assertions.assertEquals(5, calculate.numsSum(userInput));
+
+    }
+    @Test
+    public void numsMinusCheck() {
+        Mockito.when(userInput.getFirstNum()).thenReturn(2.0);
+        Mockito.when(userInput.getSecondNum()).thenReturn(3.0);
+        double result = calculate.numsDivis(userInput);
+        Assertions.assertTrue(result > 0);
```

Comment:
Not entirely correct test, if the method returns 10, then the test will pass, but 2/3 != 10
Assertions.assertEquals(result, 0.66, 0.01);